src/python/twitter/common/reviewboard/reviewboard.py

```diff
@@ -103,7 +103,7 @@ def _add_auth_params(self, username, password):
     urllib2.install_opener(opener)
 
   def get_url(self, rb_id):
-    return '%s/r/%s' % (self.url, rb_id)
+    return '%sr/%s' % (self.url, rb_id)
```

Comment:
In what way are the rules of url joins different from path joins, when you're trying to join an element to the end of a url? In what instance do you see this failing?